plasmapy/physics/tests/test_transport.py

```diff
@@ -56,3 +69,297 @@ def test_Coulomb_logarithm():
+                        decimal_places) == expected)
+
+
+# test class for _nondim_tc_i_braginskii function:
+class Test__nondim_tc_i_braginskii(object):
+    
+    def setup_method(self):
+        """set up some initial values for tests"""
+        self.big_hall = 1000
+            
+    def test_known_values_par(self):
+        kappa_i_hat = _nondim_tc_i_braginskii(self.big_hall, 
+                                              field_orientation='par') 
+        expected = 3.9 # Braginskii '65 eq (2.15)
+        decimal_places = count_decimal_places(str(expected))
+        assert(np.round(kappa_i_hat, decimal_places) == expected)
+        
+    def test_known_values_perp(self):
+        kappa_i_hat = _nondim_tc_i_braginskii(self.big_hall, 
+                                              field_orientation='perp')
+        expected = 2.0 # Braginskii '65 eq (2.16)
+        decimal_places = count_decimal_places(str(expected))
+        assert(np.round(kappa_i_hat * self.big_hall ** 2, 
+                        decimal_places) == expected)
+
+
+# test class for _nondim_tec_braginskii function:
+class Test__nondim_tec_braginskii(object):
+    
+    def setup_method(self):
+        """set up some initial values for tests"""
+        self.big_hall = 1000
+        
+    # values from Braginskii '65
+    @pytest.mark.parametrize("Z, field_orientation, expected", [
+        (1, 'par', 0.71), # eq (2.9), table 1
+        (2, 'par', 0.9),  # eq (2.9), table 1
+        (3, 'par', 1.0),  # eq (2.9), table 1
+        (4, 'par', 1.1),  # eq (2.9), table 1
+        (np.inf, 'par', 1.5),  # eq (2.9),table 1
+    ])
+    def test_known_values_par(self, Z, field_orientation, expected):
+        beta_hat = _nondim_tec_braginskii(self.big_hall, 
+                                              Z, 
+                                              field_orientation) 
+        decimal_places = count_decimal_places(str(expected))
+        assert(np.round(beta_hat, decimal_places) == expected)
+
+
+# test class for _nondim_resist_braginskii function:
+class Test__nondim_resist_braginskii(object):
+    
+    def setup_method(self):
+        """set up some initial values for tests"""
+        self.big_hall = 1000
+        
+    # values from Braginskii '65
+    @pytest.mark.parametrize("Z, field_orientation, expected", [
+        (1, 'par', 0.51), # eq (2.8), table 1
+        pytest.param(2, 'par', 0.44, marks=pytest.mark.xfail),  # table 1
+        (3, 'par', 0.40),  # eq (2.8), table 1
+        (4, 'par', 0.38),  # eq (2.8), table 1
+        (np.inf, 'par', 0.29),  # eq (2.8),table 1
+    ])
+    def test_known_values_par(self, Z, field_orientation, expected):
+        beta_hat = _nondim_resist_braginskii(self.big_hall, 
+                                              Z, 
+                                              field_orientation) 
+        decimal_places = count_decimal_places(str(expected))
+        assert(np.round(beta_hat, decimal_places) == expected)
```

Comment:
You could probably use `np.isclose(beta_hat, expected, atol = absolute_acceptable_difference_like_1e-8_or_something * units.whatever_units_we_want_here)`